.github/workflows/integration-tests.yml

```diff
@@ -84,13 +84,35 @@ jobs:
       job-tag: ${{ inputs.job-tag }}
     secrets: inherit
 
+  s390x-integration-tests:
+    uses: ./.github/workflows/integration-tests-vm-type.yml
+    if: contains(github.event.pull_request.labels.*.name, 'run-multiarch-builds') || github.event_name == 'push'
```

Comment:
As discussed, it would be interesting to limit the duplication of the build conditions along the workflow. We could make use of the architectures available in the image, as you suggested @Stringy .

Reply:
I have done some experimenting with this in another PR - https://github.com/stackrox/collector/pull/1483 

It seems to work well, but we can merge it after this PR is wrapped up